### ISSUE TITLE
Support for Signed Exchanges (SGX) - webpkgcache.com

### ISSUE THREAD

**Issue by jphorn** [piwik/piwik#20919]
Would love to see support for recognising Google's Signed Exchanges URLs in the various overviews, but especially the Acquisition reports.

## Summary
From Google: [Get started with signed exchanges on Google Search](https://developers.google.com/search/docs/appearance/signed-exchange)

> Signed exchanges (SXG) allow Google Search to prefetch your content while preserving the user’s privacy. In practice, this means that both AMP and non-AMP results shown on Google Search may prefetch a few key resources (such as HTML, JavaScript, CSS, images, or fonts) in a privacy-preserving manner, if the associated website supports SXG.
> 
> When the user ultimately clicks the result, the web page starts rendering much sooner since key resources are already available, leading to a better user experience. This could mean a lower Largest Contentful Paint (LCP) score for your content. While Google Search doesn't consider use of SXG as a direct factor in ranking, the lower LCP may affect ranking because page experience is a ranking factor.

I believe these visits should be recognised and reported as coming from channel "Search Engines" instead of "Websites" where they are currently lumped in to. Please see my screenshot.

![CleanShot 2023-06-22 at 15 14 35@2x](https://github.com/matomo-org/matomo/assets/478156/7bbf2b43-48e2-497a-8c0f-7927d86deca0)

Enabling this for your website is fairly easy for websites hosted/powered by Cloudflare ([reference](https://blog.cloudflare.com/automatic-signed-exchanges/)), so in theory a lot of websites could benefit from better insight in their acquisition traffic. 

## Your Environment

* Matomo Version: 4.14.2
* PHP Version: 8.2.5
* Server Operating System: Ubuntu 18.04
* Additionally installed plugins: BraveDetector, ReferrersManager

**Comment by michalkleiner** [comment]
Hi @jphorn, thank you for the suggestion!
It should be already possible to manage the list of search engines using Stefan's plugin https://plugins.matomo.org/ReferrersManager. Do you want to give that a go and let us know if it meets your needs?

**Comment by michalkleiner** [comment]
I'll also move this to the PO team for prioritisation on the decision whether it should become part of the core list or not.

**Comment by jphorn** [comment]
> Hi @jphorn, thank you for the suggestion!
> It should be already possible to manage the list of search engines using Stefan's plugin https://plugins.matomo.org/ReferrersManager. Do you want to give that a go and let us know if it meets your needs?

Yes. I was planning to do so, but this is more a suggestion for core. So thank you for passing it on.

**Comment by sgiehl** [comment]
@jphorn The list Matomo is using is managed in this repo: https://github.com/matomo-org/searchengine-and-social-list
Feel free to create a PR to directly add that to the list of Google's domains.
